src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -740,48 +723,6 @@ Real SpatialPooler::avgColumnsPerInput_() const {
 }
 
 
-Real SpatialPooler::avgConnectedSpanForColumn1D_(UInt column) const {
```

Comment:
You need to also remove the def for 1D/2D from header

Reply:
Thanks @breznak, I had no idea why this was failing!